W 12430553 logging combine third party gr (#196)
mulesoft/docs-gateway: gateway/1.4/modules/ROOT/pages/flex-local-view-logs.adoc

= Viewing Logs
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images

> Flex Gateway maintains standard output logs named `flex-gateway-fluent`. Use the `flex-gateway-fluent` standard output logs for quick logging checks where an additional logging output is not needed, such as ensuring an external service is running or API requests are reaching Flex Gateway.

Reviewer: see previous comment:
https://github.com/mulesoft/docs-gateway/pull/196/files#r1132828654

> Flex Gateway maintains standard output logs named `flex-gateway-fluent`. Use the `flex-gateway-fluent` standard output logs for quick logging checks where an additional logging output is not needed, such as ensuring an external service is running or API requests are reaching Flex Gateway.

Reviewer 2: I'm not sure why `flex-gateway-fluent` is the only one mentioned
```suggestion
Flex Gateway maintains standard output logs. Use the standard output logs for quick logging checks where an additional logging output is not needed, such as ensuring an external service is running or API requests are reaching Flex Gateway.
```

> You do not need to configure log output to `flex-gateway-fluent` for them to be visible. However, if you want Access Logs to appear in `flex-gateway-fluent` logs, you must apply a Message Logging policy to the APIs you want to monitor. To apply a Message Logging policy, see xref:flex-local-third-party-logs-config.adoc#configure-access-logs[Configuring Access logs].

Reviewer: What does "them" refer to? Should that be "logs" instead?

Reviewer: Is Access Logs a proper noun? If not, lowercase.

> As well as the `flex-gateway-fluent` logs, you can configure additional Fluent Bit logging outputs. Configuring additional logging enables you to apply a custom logging output and customize the log level info. To configure additional logging, see xref:flex-local-third-party-logs-config.adoc[].

Reviewer: I think the first phrase is not necessary.

> As well as the `flex-gateway-fluent` logs, you can configure additional Fluent Bit logging outputs. Configuring additional logging enables you to apply a custom logging output and customize the log level info. To configure additional logging, see xref:flex-local-third-party-logs-config.adoc[].

Reviewer: ```suggestion
Configure additional Fluent Bit logging outputs to enable you to apply a custom logging output and customize the log level info. To configure additional logging, see xref:flex-local-third-party-logs-config.adoc[].
```

> As well as the `flex-gateway-fluent` logs, you can configure additional Fluent Bit logging outputs. Configuring additional logging enables you to apply a custom logging output and customize the log level info. To configure additional logging, see xref:flex-local-third-party-logs-config.adoc[].

Reviewer 2: I'm not sure why `flex-gateway-fluent` is mentioned here

> To view additional log outputs, find the appropriate method to view them. The following commands only display the `flex-gateway-fluent` logs.

Reviewer: ```suggestion
To view additional log outputs, find the appropriate method to view them. The following commands display only the `flex-gateway-fluent` logs.
```

Reviewer: https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22only%22(%22simply,%22%22just%22)

> To view additional log outputs, find the appropriate method to view them. The following commands only display the `flex-gateway-fluent` logs.

Reviewer 2: This display all logs though

== View Logs

> To view the standard output logs, run the following commands:

Reviewer: I assume they run one command, not all. 
Maybe "run one of the following commands"?

For Linux:
[source,ssh]
----
journalctl -u flex-gateway
----

For Docker:
[source,ssh]
----
docker logs -f <container-id>
----

For Kubernetes:
[source,ssh]
----
kubectl logs -f <pod-name>
----

== See Also

> * https://docs.fluentbit.io/manual/v/1.8/pipeline/outputs[Fluent Bit Outputs documentation ^]

Reviewer 3: Could you use your new antora var to construct this URL?
